### PR TITLE
KY Job Listing 

### PR DESCRIPTION
**Pull request by kiyonggg**

### CODE REVIEW

File: src/components/KYJobListing/styles/JobListingCard.styled.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components'
+
+//media: set flex-direction to col
```
Comment (xinyee20) [inline src/components/KYJobListing/styles/JobListingCard.styled.ts]: can be removed

File: src/components/KYJobListing/styles/JobListingCard.styled.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  margin: 30px;
+  padding: 20px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 5px 10px 10px #dedede;
+  ${(props) =>
+    props.active &&
+    `
+    border-left: 5px solid hsl(180, 29%, 50%);
+  `}
```
Comment (xinyee20) [inline src/components/KYJobListing/styles/JobListingCard.styled.ts]: the formatting is a bit weird, they can all be put in the same line in this case

File: src/components/KYJobListing/styles/JobTag.styled.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  font-family: 'Spartan', sans-serif;
+  font-weight: 700;
+  font-size: 9px;
+  color: white;
+  margin: 0;
+  height: min-content;
+`
+// position: absolute;
+// top: 50%;
+// transform: translateY(-50%);
```
Comment (xinyee20) [inline src/components/KYJobListing/styles/JobTag.styled.ts]: comments can be removed

File: src/pages/FrontendMentor/KYJobListing/index.tsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  FULL_TIME = 'Full Time',
+  PART_TIME = 'Part Time',
+  CONTRACT = 'Contract',
+}
+// company: string
+// jobTitle: string
+// tags: JobTagEnum[]
+// time: string
+// jobType: JobTypeEnum
+// location: string
```
Comment (xinyee20) [inline src/pages/FrontendMentor/KYJobListing/index.tsx]: comments can be removed

Comment (kiyonggg) [reply src/pages/FrontendMentor/KYJobListing/index.tsx]: resolved in commit e616b1a

File: src/components/KYJobListing/styles/JobTag.styled.ts (hunk 3 above)
Comment (kiyonggg) [reply src/components/KYJobListing/styles/JobTag.styled.ts]: resolved in commit e616b1a

File: src/components/KYJobListing/styles/JobListingCard.styled.ts (hunk 2 above)
Comment (kiyonggg) [reply src/components/KYJobListing/styles/JobListingCard.styled.ts]: resolved in commit e616b1a

File: src/components/KYJobListing/styles/JobListingCard.styled.ts (hunk 1 above)
Comment (kiyonggg) [reply src/components/KYJobListing/styles/JobListingCard.styled.ts]: resolved in commit e616b1a

**Review by xinyee20 — APPROVED** [review]
Lgtm!

**Pull request merged**